CHANGELOG

```diff
@@ -1,6 +1,9 @@
 devel
 -----
 
+* Added ENTRIES function, that takes an object and produces a list of key-value
+  pairs.
```

Comment:
I suggest that we add here that the order in which attributes are iterated is not guaranteed and thus is an implementation detail.